translations/es.ts

```diff
@@ -4,7 +4,7 @@
 <context>
     <name>About</name>
     <message>
-        <source>OSM Scout for Sailfish OS is developed as open-source by volunteers in their free time. You can help to improve this application by reporting bugs, creating translations or developing new features. Any help is welcome.</source>
+        <source> OSM Scout for Sailfish OS is developed as open-source by volunteers in their free time. You can help to improve this application by reporting bugs, creating translations or developing new features. Any help is welcome.</source>
```

Comment:
This space at the beginning of the phrase seems to be error... Strings have to match for processing with lupdate and lrelease tools.